oauth_manager.go

```diff
@@ -384,13 +385,12 @@ type ExtendedOsinStorageInterface interface {
 	// Custom getter to handle prefixing issues in Redis
 	GetClientNoPrefix(id string) (osin.Client, error)
 
+	GetClientTokens(id string, page int) ([]OAuthClientToken, float64, error)
```

Comment:
I'd prefer new method leaving existing one as is retrieving all tokens from redis, something like `GetClientTokensPaginated`

Reply:
I went with `GetPaginatedClientTokens`.